projects/make-recall-decisions-and-delius/src/main/kotlin/uk/gov/justice/digital/hmpps/integrations/delius/casesummary/Address.kt

```diff
@@ -10,14 +10,15 @@ import jakarta.persistence.Table
 import org.hibernate.annotations.Immutable
 import org.hibernate.annotations.Where
 import org.hibernate.type.YesNoConverter
+import org.springframework.data.jpa.repository.EntityGraph
 import org.springframework.data.jpa.repository.JpaRepository
 import uk.gov.justice.digital.hmpps.integrations.delius.referencedata.entity.ReferenceData
 
-@Entity
 @Immutable
 @Table(name = "offender_address")
+@Entity(name = "CaseSummaryAddress")
```

Comment:
Yeah we only need a different entity name so that JPA doesn't conflict with the existing recommendation entities.  This way we don't have to repeat the redundant CaseSummary* prefix all over the actual code - we just import the Person/Address/whatever from the casesummary package.